packages/openactive-integration-tests/test/features/approval/minimal-proposal/implemented/accept-proposal-book-test.js

```diff
@@ -39,6 +39,7 @@ FeatureHelper.describeFeature(module, {
   testOpportunityCriteria: 'TestOpportunityBookableFlowRequirementOnlyApproval',
   // even if some OrderItems don't require approval, the whole Order should
   controlOpportunityCriteria: 'TestOpportunityBookable',
+  skipBookingFlows: ['OpenBookingSimpleFlow'],
```

Comment:
These approval flows now slot nicely into the framework. There's an outstanding TODO to remove the `TestOpportunityBookableFlowRequirementOnlyApproval` criteria, which is now redundant (as it is the same as `TestOpportunityBookable` x `OpenBookingApprovalFlow`)

Reply:
I did that earlier actually